dataline-integrations/singer/exchangerateapi_io/source/build.gradle

```diff
@@ -35,6 +35,8 @@ task imageName {
 task buildImage(type: DockerBuildImage) {
     inputDir = projectDir
     images.add(image)
+
+    dependsOn(':dataline-integrations:singer:base:buildImage')
```

Comment:
We'll want to add this to all downstream integration builds so we make sure we're using an up to date base image.